samples/DataSourceWithProcessingAndExtensions/DataGenerator/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace DataGenerator
+{
+    class Program
```

Comment:
>class Program [](start = 4, length = 13)

Can we add a comment somewhere for this entire project explaining what it's for and how it's used? Just to be thorough in our documentation for samples